Dockerfile

```diff
@@ -21,11 +21,16 @@ ARG GOOS=linux
 # ARG GOARCH=amd64
 
 RUN go build -o /adam/bin/adam main.go
+RUN go get -u github.com/go-swagger/go-swagger/cmd/swagger
+RUN GO111MODULE=off swagger generate spec -o ./swaggerui/swagger.json
```

Comment:
It looks like this generates `swaggerui/swagger.json`, but then this is never copied from the build image to the final image.

Reply:
Yes I removed it